### PR TITLE
Improve front matter

### PR DESCRIPTION
**Pull request by marians**
Replacement for https://github.com/giantswarm/docs/pull/805 with a more manageable amount of changes

### Please remember to

- [x] Give the PR a meaningful title -- it will be shown in the release notes
- [ ] Add (or remove) [user questions](https://github.com/giantswarm/docs/blob/master/CONTRIBUTING.md#front-matter) associated with any updated docs

### CODE REVIEW

File: src/data/changes_categories.yaml [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 Rest API: '#919ea1'
 Web UI: '#8098b2'
-Workload Cluster Releases for AWS: '#ff9900'
-Workload Cluster Releases for Azure: '#1773bd'
-Workload Cluster Releases for KVM: '#cccccc'
+Workload cluster releases for AWS: '#ff9900'
+Workload cluster releases for Azure: '#1773bd'
+Workload cluster releases for KVM: '#cccccc'
 gsctl: '#7cb2b6'
 kubectl gs: '#8081b2'
```
Comment (marians) [inline src/data/changes_categories.yaml:12]: While this file should actually not be changed manually, it also doesn't hurt.

File: src/content/advanced/custom-admission-controller/index.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 linkTitle: Custom admission controllers
 title: Creating your own admission controller
-description: Tutorial on how to create your own admission controller.
+description: Tutorial on how to create your own admission controller, to enforce certain custom rules and policies in the resources created in your Kubernetes cluster.
```
Comment (stone-z) [inline src/content/advanced/custom-admission-controller/index.md:4]:
```suggestion
description: Tutorial on how to create your own admission controller to enforce custom rules and policies in the resources created in your Kubernetes cluster.
```

File: src/content/advanced/high-availability/multi-az/index.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 ---
+linkTitle: Multiple AZ
 title: Clusters over multiple availability zones
-description: Using multiple availability zones both for worker and for master nodes increases the resilience for clusters. Here we explain some details regarding support on different cloud providers and in different workload cluster releases. And we give basic information on how to configure workloads to leverage multiple availability zones.
+description: Using multiple availability zones both for worker and for master nodes increases the resilience for clusters. Here we explain some details regarding support on different cloud providers and releases, plus how to configure workloads to leverage multiple availability zones.
```
Comment (stone-z) [inline src/content/advanced/high-availability/multi-az/index.md:4]:
```suggestion
description: Using multiple availability zones both for worker and for master nodes increases the resilience of the cluster. Here we explain some details regarding support on different cloud providers and releases, plus how to configure workloads to leverage multiple availability zones.
```

File: src/content/general/support/index.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ---
 linkTitle: Support
 title: Giant Swarm support
-description: An explanation of how our support service works.
+description: An explanation of how our support service and various specific processes and workflows.
```
Comment (stone-z) [inline src/content/general/support/index.md:4]: The new version needs another verb (or to drop "how"), or simply:
```suggestion
description: An explanation of our support service processes and workflows.
```

File: src/content/ui-api/gsctl/list-keypairs.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 ---
 linkTitle: list keypairs
 title: "'gsctl list keypairs' command reference"
-description: "The 'gsctl list keypairs' lists key pairs for a cluster."
+description: "Dokcumentation of the 'gsctl list keypairs' command, which prints a list of all key pairs currently valid for accessing a workload cluster."
```
Comment (stone-z) [inline src/content/ui-api/gsctl/list-keypairs.md:4]:
```suggestion
description: "Documentation of the 'gsctl list keypairs' command, which prints a list of all key pairs currently valid for accessing a workload cluster."
```

Comment (stone-z) [reply src/content/ui-api/gsctl/list-keypairs.md:4]: This has a different style than most other command descriptions. `delete nodepool` also has a different style.
Most commands follow the "The ___ command shows..." style

File: src/content/ui-api/gsctl/login.md [hunk 6]
```diff
@@ -1,7 +1,7 @@
 ---
 linkTitle: login
 title: "'gsctl login' command reference"
-description: "The 'gsctl login' command starts an authenticated session."
+description: "The 'gsctl login' command ensures an authenticated session with a Giant Swarm installation's Rest API."
```
Comment (stone-z) [inline src/content/ui-api/gsctl/login.md:4]:
```suggestion
description: "The 'gsctl login' command ensures an authenticated session with a Giant Swarm installation's REST API."
```

**Review by stone-z — APPROVED** [review]
Some comments but generally LGTM

File: scripts/aggregate-changelogs/script.py [hunk 7]
```diff
@@ -227,15 +227,15 @@ def generate_release_file(repo_shortname, repo_config, release):
...
+        categories = [f'Workload cluster releases for {provider_label}']
+        title = f'Workload cluster release v{version} for {provider_label}'
+        description = f'Release notes for {provider_label} workload cluster release v{version}, published on {release["date"].strftime("%d %B %Y, %H:%M")}'
         filename = f"{release['provider']}-{release['version_tag']}.md"
         category_path = f"workload-cluster-releases-{provider_label.lower()}"
         aliases = [f"/changes/tenant-cluster-releases-{provider_label.lower()}/releases/{provider_label.lower()}-{release['version_tag']}/"]
     else:
         categories = [repo_config['category']]
-        title = f'{repo_id} Release v{version}'
+        title = f'{repo_id} release v{version}'
```
[the remark this answers is not in the document]
Comment (marians) [reply scripts/aggregate-changelogs/script.py:238]: It ends up in titles like for this page: https://docs.giantswarm.io/changes/managed-apps/nginx-ingress-controller-app/v1.14.1/

We decided we want sentence case for headlines, so I'm adapting that here.

File: src/content/general/support/index.md (hunk 4 above)
Comment (marians) [reply src/content/general/support/index.md:4]: Hmm. We need something longer :o

File: scripts/aggregate-changelogs/script.py (hunk 7 above)
[the remark this answers is not in the document]
Comment (stone-z) [reply scripts/aggregate-changelogs/script.py:238]: Oh yes, in the middle of the title like that it definitely makes sense to lowercase it

File: src/content/general/support/index.md (hunk 4 above)
Comment (stone-z) [reply src/content/general/support/index.md:4]: How much longer? Would saying "various support service processes and workflows" make it?

**Comment by marians** [comment]
Merging as best-effort

**Pull request merged**